Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.13 as builder
+
+WORKDIR /app
+COPY . /app
+
+RUN go get -d -v
```

Comment:
any differences between `go mod vendor` and `go get -d` to pull dependencies?

Reply:
seems like vendor will download and make copies into a vendor directory, go get will just upgrade versions in go.mod. do we need to vendor?

Reply:
yes I think so. Can you patch the previous PR I merged in as well? O copied the `go get -d`

Reply:
done

Reply:
go mod vendor requires `go build -mod=vendor`. So it would be a little redundant otherwise. `go mod tidy` may be what we're looking for here.

Reply:
ah gotcha! My understanding is that both download the dependencies locally, vendor makes puts it in `vendor/`, while `go get` leaves them in `$GOPATH/pkg`. 

I honestly don't have a preference / don't know enough detail to pick out the nuance. So am fine with either.

Reply:
let's create an issue to verify this later and confirm with the golang team. @asraa already moved to using `go vendor` for now, we'll change if the other option is better after that.